app/services/container_topology_service.rb

```diff
@@ -89,22 +89,26 @@ def entity_display_type(entity)
     end
   end
 
-  def build_entity_data(entity)
+  def entity_id(entity)
     type = entity_type(entity)
-    status = entity_status(entity, type)
-
     if entity.kind_of?(ManageIQ::Providers::ContainerManager)
       id = entity.id.to_s
```

Comment:
> Why does it need to be a string? If used in links, can't we use string interpolation so we can pass the id as is?

not sure how string interpolation helps here? all other ids passed here are strings, and the output later is converted to json and passed to js